docs/get-started/testnets-and-devnets.md

```diff
@@ -7,39 +7,60 @@ image: ../img/og/og-getstarted-testnets.png
 --- 
 
 To make the difference between Cardano mainnet functionality and testnet functionality clearer, we moved the old content of [developers.cardano.org](https://developers.cardano.org) to [testnets.cardano.org](https://testnets.cardano.org) with the launch of this developer portal.
-
 ## Cardano testnet
 The [Cardano testnet](https://testnets.cardano.org/en/testnets/cardano/overview/) is your playground when testing [Cardano integration](/docs/integrate-cardano/), building with [transaction metadata](/docs/transaction-metadata/), exploring [native tokens](/docs/native-tokens/) or learning how to [operate a stake pool](/docs/operate-a-stake-pool/).
 
-Because the [Cardano testnet](https://testnets.cardano.org/en/testnets/cardano/overview/) is an independent network, separate from the Cardano mainnet, it requires its own token: test ada (tAda).  
+Testnet is an essential part of the development process for Cardano, as it allows developers to test and refine their code before deploying it on the live network, ultimately improving the reliability and security.
 
 ### Where to get test ada?
+
+Because the [Cardano testnet](https://docs.cardano.org/cardano-testnet/overview/) is an independent network, separate from the Cardano mainnet, it requires its own token: test ada (tAda).
```

Comment:
A bit of knitpicking, but:
1. I think in general we should _start_ by clarifying testnet contains multiple networks , and use `testnets` (plural) where possible.
2. The hyperlink reference here is good for resources (mainly faucet and Plutip), but it does repeat a lot of information that's already on dev portal, and when the target site isnt up-to-date - can become a liability (besides, it's a little confusing to have multiple 'Installing node'/'Downloading config'/etc sections across sites.

IMO - it could be better to adopt those 2 resource links later on this page and get rid of external reference to this particular site completely, and simply have single reference (already the case below against networks) that's directly maintained and used by IO devops teams themselves (thus, can be sure it's almost always going to be up-to-date).

Reply:
Re


> 1. I think in general we should _start_ by clarifying testnet contains multiple networks , and use `testnets` (plural) where possible.

I agree on this one, but after talking to some people, I've heard many users use this page to find test ada information so I bumped on top to save users some time.

> 2. The hyperlink reference here is good for resources (mainly faucet and Plutip), but it does repeat a lot of information that's already on dev portal, and when the target site isnt up-to-date - can become a liability (besides, it's a little confusing to have multiple 'Installing node'/'Downloading config'/etc sections across sites).

So if I understand correctly - redirect internally on DP on faucet and Plutip pages?

Reply:
Seems above questions are still unresolved, but resolving this thread because otherwise it blocks merging the more basic information that needs to be released... we still get very basic questions about the Testnets from the community at large, which are answered here.